addons/project/tests/test_project_sharing_ui.py

```diff
@@ -37,6 +37,66 @@ def setUpClass(cls):
             ],
         })
 
+    def test_blocked_by_project_sharing_string_portal(self):
+
+        user_group_project_manager = self.env.ref('project.group_project_manager')
+        user_group_employee = self.env.ref('base.group_user')
```

Comment:
group_user = self.env.ref('base.group_user')
```suggestion
         group_user = self.env.ref('base.group_user')
```